level2/instance.tf

```diff
@@ -70,9 +69,10 @@ resource "aws_instance" "private" {
   instance_type          = "t3.micro"
   key_name               = "devops"
   vpc_security_group_ids = [aws_security_group.private.id]
-  subnet_id              = data.terraform_remote_state.level1.outputs.private_subnet_id[0]
+  subnet_id              = data.terraform_remote_state.level1.outputs.private_subnet_id[1]
```

Comment:
Add count variable and change` data.terraform_remote_state.level1.outputs.private_subnet_id[1]` to `data.terraform_remote_state.level1.outputs.private_subnet_id[count.index]`